Sources/Typology/AST/SwiftSyntax.swift

```diff
@@ -39,11 +40,11 @@ extension Array where Element == Statement {
                 )
               },
             statements: body.flatMap([Statement].init) ?? [],
-            returns: returns.map(Type.init) ?? .tuple([])
+            returns: returns.map(Type.init) ?? .tuple([]), position: position ?? AbsolutePosition(line: 1, column: 1, utf8Offset: 0)
```

Comment:
I don't think we should ever use a default position of line 1, column 1 like that, as this is a meaningless position. If we're absolutely sure that the we need to operate on AST nodes without a position, we should make `position` optional and use `nil` as a default. In this case it won't be needed if you use `function.position` instead of `function.body?.position`